source/installation.txt

```diff
@@ -23,6 +23,15 @@ MongoDB.
 
 For upgrade instructions, see :ref:`3.6-upgrade` instead.
 
+.. note::
+
+   .. versionadded:: 3.6
+
+      Each MongoDB installer is packaged with a platform-specific
+      :ref:`MongoDB Compass <compass-index>` installation script. This
+      script installs MongoDB Compass as part of the MongoDB Server
+      installation process.
```

Comment:
we'll actually need to include a install_compass for the installation script under
https://docs.mongodb.com/master/reference/program/

Then, basically, this new page just state that this is the installer for Compass Community edition and how to run.

Then link to: https://docs.mongodb.com/compass/current/#compass-and-compass-community-editions